.storybook/stories/templates/01-Login-page.stories.tsx

```diff
@@ -7,42 +7,34 @@ import { HeaderLogic, HeaderLogicProps } from '../../../components/Header/src';
 import Footer from '../../../components/Footer/src';
 import ResponsiveContent from '../../../components/ResponsiveContent/src';
 import { AuthenticationCardGroup } from '../../../components/CardGroup/src/story-template';
-import {
-  headerProps,
-  footerLegalData,
-  footerSocialData,
-  newsletterData,
-  contactData,
-  copyright,
-} from '../templates/util';
+import { headerProps, footerLegalData, footerSocialData, newsletterData, contactData, copyright } from './util';
 
-import './story-login.css';
+import './template-story.scss';
 
 const meta = {
-  title: 'Templates',
+  title: 'Templates/LoginPagina',
```

Comment:
met de /LoginPagina kom die toch in een mapje in de storybook? ik dacht dat we dat juist niet wilden,  of toch wel omdat het er mooier uitziet in de code?

Reply:
Goed gespot!

Als ze precies hetzelfde heten komen ze ook niet in een mapje en is de header beter. Viel mij op dat hij er raar uitzag zoals ik eerder had voorgesteld dus heb even zitten debuggen ;)

Reply:
Dus x.stories.tsx met 1 story erin kan een title hebben en een story met precies de naam na de `/` en dan werkt het toppie

Reply:
Heeft een bestand meerdere stories, dan geven we elke story de naam van wat hij doet. Dus bijvoorbeeld
`title="CSS/Button"` met een `const PrimaryAction = {...}` en een `const SecondaryAction = {...}`